contracts/vortex/CarbonVortex.sol

```diff
@@ -81,7 +125,22 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
      * @dev performs contract-specific initialization
      */
     function __CarbonVortex_init_unchained() internal onlyInitializing {
-        _setRewardsPPM(100_000);
+        // set rewards ppm to 5000
```

Comment:
`PPM`?

Reply:
points per million

Reply:
I meant that it should be "PPM" and not "ppm"

Reply:
fixed